app/src/main/java/org/sagebase/crf/step/CrfInstructionStep.java

```diff
@@ -74,6 +78,12 @@
      */
     public boolean mediaVolume;
 
+    /**
+     * If true, the remind me later button will show, if false it will hide, defaults to true
+     */
+    public boolean remindMeLater;
```

Comment:
Should this move with the rest of the reminder stuff to CrfSkipInstructionStep?

Reply:
Just moved it over!

Reply:
Moved